src/pages/home-page-slides/home-page-slides-box-text-mobile/home-page-slides-box-text-mobile.tsx

```diff
@@ -0,0 +1,41 @@
+import { Dispatch, FC } from "react";
+import {
+  BoxTextMobile,
+  SmallParagraph,
+  MainParagraph,
+  ButtonBoxText,
+} from "./home-page-slides-box-text-mobile.style";
+import ButtonsSlider from "../home-page-slides-button-slider/buttons-slider";
+import { ButtonsBox } from "../home-page-slides-mobile-width.style";
+import { CarouselModel } from "../../../model/model";
+
+interface HomePageSlidesBoxTextMobilePropsModel {
```

Comment:
You can omit the ending "model" when naming models, types etc

Reply:
ok, but I don't know how to name individual interfaces and types because I haven't called them like that before and this was also an error noticed by Michał.